app/representation.py

```diff
@@ -3,37 +3,44 @@ def venueRecord(biz, **details):
     providers = {}
     images = []
     hours = []
+    categories = set()
     if "yelp" in details:
       info = details["yelp"]
       providers["yelp"] = {
         "rating": biz.rating,
-        "reviewCount": biz.review_count,
+        "totalReviewCount": biz.review_count,
         "ratingMax": 5,
-        "description": biz.snippet_text,
+        "summary": biz.snippet_text,
         "url": biz.url
       }
       images = images + info["photos"]
       hours = info["hours"][0]["open"]
+      categories |= set([ c["title"] for c in info["categories"] if "title" in c])
```

Comment:
Does Yelp provide categories sorted by any priority? If so, I think it'd be great to preserve that priority on the client.